.changeset/blood-moon-rising.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fluid-experimental/property-query": minor
+---
+
+`joi` dependency was updated from 14.3.1 to 17.3.1 to address a critical vulnerability exploit [CVE-2020-36604](https://github.com/advisories/GHSA-c429-5p7v-vgjp). This required updating the use of `joi` schema validation function within `property-query` to the new major version syntax.
```

Comment:
These normally have a change summary line, and then the details, so the release notes can use the summary as title/header entries.
```suggestion
Updated `joi` dependency to latest major version

The `joi` dependency was updated from 14.3.1 to 17.3.1 to address a critical vulnerability exploit [CVE-2020-36604](https://github.com/advisories/GHSA-c429-5p7v-vgjp). This required updating the use of `joi` schema validation function within `property-query` to the new major version syntax.
```